var/spack/repos/builtin/packages/py-pillow/package.py

```diff
@@ -70,7 +75,7 @@ def patch(self):
 
         def variant_to_cfg(variant):
             able = "enable" if "+" + variant in self.spec else "disable"
-            return "{0}-{1}=1\n".format(able, variant)
+            return "{0}_{1}=1\n".format(able, variant)
```

Comment:
setuptools 54.1+ issues a warning about using `enable-foo`, we should instead be using `enable_foo`. I didn't test ancient setuptools to make sure that this works.